### ISSUE TITLE
Windows version, edition, service pack are missing in inventory

### ISSUE THREAD

**Issue by eric95630** [glpi-project/glpi#11367]
### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### Is there an existing issue for this?

- [X] I have searched the existing issues

### Version

10.0 (final)

### Bug description

Since the update of GLPI, the inventory of my machines does not show the version, the service pack, nor the edition. The problem is identical with the FusionInventory agent or the GLPI agent installed on the machines.

![image](https://user-images.githubusercontent.com/104144568/164735977-0b9fca43-c627-4d2b-90ec-1db2880029d8.png)

The first line is an inventory with GLPI 10, the 2nd line with GLPI 9

**Comment by trasher** [comment]
Please provide an XML inventory file to reproduce.

**Comment by eric95630** [comment]
Is its possible to send it in private ? Because it contains private datas.

**Comment by trasher** [comment]
This is included in professional support. 

You can limit inventory to only relevant parts (see agents docs) and anonymize data on your side before sending.

**Comment by eric95630** [comment]
> This is included in professional support.
> 
> You can limit inventory to only relevant parts (see agents docs) and anonymize data on your side before sending.

Here it is. Thank
This an example with another server. Same problem.
[computer_0_417.json.zip](https://github.com/glpi-project/glpi/files/8562330/computer_0_417.json.zip)
s

**Comment by eric95630** [comment]
Thanks for the feedback.
Actually, I don't really understand what you're telling me. 
I did a test with the GLPI Agent and the Fusioninventory Agent, the result is the same. For the machine with the FusionInventory agent, I just updated the access URL to the GLPI server.
Before the update of GLPI to version 10, I had the details of the OS (name, version, service pack, edition...). It was important for me for the inventories of parks and the follow-up of updates.
This is not the case anymore.
